### DISCUSSION TITLE
Apollo server deploy example

### DISCUSSION THREAD

**Discussion by saihaj** [vercel/vercel#4792] [Help] [answered]
How can I deploy an Apollo GraphQL server to Vercel? I saw an old example https://github.com/lukeburns/now-examples/tree/master/apollo are there any new examples?

**Comment by mcsdevv** [comment, marked as answer]
Hello @saihaj, we recommend deploying Apollo Server on alternative platforms that utilize a running server. Whilst technically possible to use it inside a Serverless Function there are limitations, as a result, we do not recommend this.
